src/RelationManagers/AuthenticationLogsRelationManager.php

```diff
@@ -31,11 +31,11 @@ public function table(Table $table): Table
                 Tables\Columns\TextColumn::make('authenticatable')
                     ->label(trans('filament-authentication-log::filament-authentication-log.column.authenticatable'))
                     ->formatStateUsing(function (?string $state, Model $record) {
-                        if (! $record->authenticatable_id) {
+                        if (!$record->authenticatable_id) {
                             return new HtmlString('&mdash;');
                         }
 
-                        return new HtmlString('<a href="'.route('filament.'.Filament::getCurrentPanel()->getId().'.resources.'.Str::plural((Str::lower(class_basename($record->authenticatable::class)))).'.edit', ['record' => $record->authenticatable_id]).'" class="inline-flex items-center justify-center hover:underline focus:outline-none focus:underline filament-tables-link text-primary-600 hover:text-primary-500 text-sm font-medium filament-tables-link-action">'.class_basename($record->authenticatable::class).'</a>');
+                        return new HtmlString('<a href="' . route('filament.' . Filament::getCurrentPanel()->getId() . '.resources.' . Str::plural((Str::lower(class_basename($record->authenticatable::class)))) . '.edit', ['record' => $record->authenticatable_id]) . '" class="inline-flex items-center justify-center hover:underline focus:outline-none focus:underline filament-tables-link text-primary-600 hover:text-primary-500 text-sm font-medium filament-tables-link-action">' . $record->authenticatable->name . '</a>');
```

Comment:
Sorry for the delay, thank you for taking the time to work on this!

What about adding a config to provide the field to display, and we keep the class name as default if this config is not provided (if `field-to-display` it's null). Something like:

On `config/filament-authentication-log.php`:

```php
    'authenticatable' => [
        'field-to-display' => null,
    ],  
```

and on `src/RelationManagers/AuthenticationLogsRelationManager.php`:

```php
$authenticatableFieldToDisplay = config('filament-authentication-log.authenticatable.field-to-display');

$authenticatableDisplay = $authenticatableFieldToDisplay !== null ? $record->authenticatable->{$authenticatableFieldToDisplay} : class_basename($record->authenticatable::class);

return new HtmlString('<a href="' . route('filament.' . Filament::getCurrentPanel()->getId() . '.resources.' . Str::plural((Str::lower(class_basename($record->authenticatable::class)))) . '.edit', ['record' => $record->authenticatable_id]) . '" class="inline-flex items-center justify-center hover:underline focus:outline-none focus:underline filament-tables-link text-primary-600 hover:text-primary-500 text-sm font-medium filament-tables-link-action">'.$authenticatableDisplay.'</a>');
```

Reply:
Good Improvement I will add, it may be today

Reply:
**Sorry for the delay.** I was involved in some other work and was not able to finish it sooner.

Now, it is completed. Please take a look at it. The README has also been updated.